presto-docs/src/main/sphinx/sql/select.rst

```diff
@@ -666,12 +668,42 @@ output expressions:
 
     ORDER BY expression [ ASC | DESC ] [ NULLS { FIRST | LAST } ] [, ...]
 
-Each expression may be composed of output columns or it may be an ordinal
-number selecting an output column by position (starting at one). The
-``ORDER BY`` clause is evaluated after any ``GROUP BY`` or ``HAVING`` clause
+Each expression may be composed of output columns, or it may be an ordinal
+number selecting an output column by position, starting at one. The
+``ORDER BY`` clause is evaluated after any ``GROUP BY`` or ``HAVING`` clause,
 and before any ``OFFSET``, ``LIMIT`` or ``FETCH FIRST`` clause.
 The default null ordering is ``NULLS LAST``, regardless of the ordering direction.
 
+Note that, following the SQL specification, an ``ORDER BY`` clause only
+affects the order of rows for queries that immediately contain the clause.
+Presto follows that specification, and drops redundant usage of the clause to
+avoid negative performance impacts.
+
+In the following example, the clause only applies to the select statement.
+
+.. code-block:: SQL
+
+    INSERT INTO some_table
+    SELECT * FROM another_table
+    ORDER BY field
+
+Since tables in SQL are inherently unordered, and the ``ORDER BY`` clause in
+this case does not result in any difference, but negatively impacts performance
+of running the overall insert statement, Presto skip the sort operation.
+
+Another example, where the ``ORDER BY`` clause is redundant, and does not affect
```

Comment:
The comma usage in this sentence makes it sound choppy.

Reply:
fixed